hack/util.sh

```diff
@@ -54,6 +46,22 @@ function setup_env_vars {
 		export CURL_CERT="${MASTER_CONFIG_DIR}/admin.crt"
 		export CURL_KEY="${MASTER_CONFIG_DIR}/admin.key"
 	fi
+}
+
+# setup_temp_dir_vars sets ups all temporary directory variables given the TMPDIR
+function setup_temp_dir_vars() {
+	local sub_dir=$1
+
+	export BASETMPDIR="${BASETMPDIR:-/tmp/openshift/origin/}"
+	export TMPDIR="${TMPDIR:-$sub_dir}"
```

Comment:
Our previous behavior was to always over-ride `$TMPDIR` with our own - should it be set to something as default? It's unset on my machine by default.